controllers/util/util.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2021 The Kubernetes authors.
```

Comment:
Please check all copyright, kubernetes should be modified to OpenYurt.

Reply:
done